packages/cli/src/util/link/handle-options.ts

```diff
@@ -0,0 +1,96 @@
+import chalk from 'chalk';
+import { Org, Project, ProjectSettings } from '../../types';
+import Client from '../client';
+import {
+  connectGitProvider,
+  disconnectGitProvider,
+  formatProvider,
+  RepoInfo,
+  parseRepoUrl,
+} from '../git/connect-git-provider';
+import { Output } from '../output';
+import { getCommandName } from '../pkg-name';
+import updateProject from '../projects/update-project';
+
+export async function handleOptions(
+  option: string,
+  client: Client,
+  org: Org,
+  project: Project,
+  settings: ProjectSettings,
+  repoInfo?: RepoInfo
+) {
+  const { output } = client;
+  if (option === 'no') {
+    return skip(output);
```

Comment:
**preference (non-blocking):** separate `return` to make it clearer that no value is expected from `skip`

```suggestion
    skip(output);
    return;
```

Skip only returns `undefined`. This code works, but it implies more is going on. I suggest separating them. The same goes for `return optOut;`.

I think this all stems from the convention of functions used by CLI commands to return a number or whatever other type they would successfully return otherwise. We should discuss and possibly refactor these out, but not in this PR or overall task.